app/services/submit_application.rb

```diff
@@ -29,6 +29,16 @@ def submit_application
     application_choices.each do |application_choice|
       application_choice.edit_by = ApplicationDates.new(application_form).edit_by
       ApplicationStateChange.new(application_choice).submit!
+      StateChangeNotifier.call(:submit_application, application_choice: application_choice)
```

Comment:
We want just 1 notification that the form has been submitted (all application_choices are submitted together).

Reply:
Ok